src/backend/distributed/sql/udfs/get_all_active_transactions/11.0-1.sql

```diff
@@ -6,7 +6,7 @@ RETURNS SETOF RECORD
 LANGUAGE C STRICT AS 'MODULE_PATHNAME',
 $$get_all_active_transactions$$;
 
-COMMENT ON FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT datname text, OUT process_id int, OUT initiator_node_identifier int4,
+COMMENT ON FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_id int, OUT initiator_node_identifier int4,
```

Comment:
should we add a brand new sql file instead ?

Reply:
Why new file? This is bug-fix in the file and not adding/deleting a new parameter. Am I missing something?

Reply:
Even when we're doing a bugfix, we still prefer to have a separate sql file to not break users that already installed given version. For example, if we do those changes in 11.0-1 schema, then users that already has this version installed wouldn't get the fix.

Reply:
Yes, you're right. This change won't even run if I just change the old sql file. I updated and added upgrade and downgrade scripts.

Reply:
> Why new file? This is a bug-fix in the file and not adding/deleting a new parameter. Am I missing something?

It is a tricky business to maintain multiple major versions. We like to introduce new files for several reasons:

1. It makes more sense to introduce new sql files that capture the history of definition changes. (See https://github.com/citusdata/citus/tree/main/src/backend/distributed/sql/udfs/citus_finish_pg_upgrade folder for one of the best example on how a UDF evolved over time.)
2. When we see the output of `\dx` we are more confident on the definitions of database objects as we do not change definitions after we release the packages for that version.
3. Migration files only get applied during an upgrade/downgrade. Say you fixed a bug on an upgrade script from `10.0` to `10.1`. It will not be applied to new clusters that already upgraded to `10.1`. In that case we need to introduce new migration versions for all the majors that were affected. We usually introduce new migration versions that are affected, instead of updating definition files in place.
4. It makes it easier to have clean documentation on what changed in patch versions.

Reply:
Thanks @halilozanakgul @onurctirtir I concur with you, but since this is a simple and harmless comment change (which shows up with \df), maybe we should simply update the file? I'm fine either way.